Sources/Storyboard/StoryboardBased.swift

```diff
@@ -28,7 +28,7 @@ public protocol StoryboardBased: class {
 public extension StoryboardBased {
   /// By default, use the storybaord with the same name as the class
   static var storyboard: UIStoryboard {
-    return UIStoryboard(name: String(self), bundle: NSBundle(forClass: self))
+    return UIStoryboard(name: String(describing: self), bundle: Bundle(for: self))
```

Comment:
`StoryboardBased` is designed for the case where you have an `initialViewController` with the same name as the Storyboard file. As it instantiate the VC using `initialViewController` (and not using a `sceneIdentifier`), having a `sceneIdentifier` for the `StoryboardBased` shouldn't be a requirement (as it's optional, and not required for the `StoryboardBased` mixin to work)